packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -126,23 +131,15 @@ describe(`Test plugin manifest options`, () => {
   it(`fails on non existing icon`, async () => {
     fs.statSync.mockReturnValueOnce({ isFile: () => false })
 
+    let pluginSpecificOptions = {
```

Comment:
This can also be a const right?
```suggestion
    const pluginSpecificOptions = {
```